src/ImageSharp/Image.FromBytes.cs

```diff
@@ -65,13 +69,16 @@ public static IImageFormat DetectFormat(Configuration configuration, byte[] data
         /// <param name="configuration">The configuration.</param>
         /// <param name="data">The byte array containing encoded image data to read the header from.</param>
         /// <param name="format">The format type of the decoded image.</param>
-        /// <exception cref="NotSupportedException">Thrown if the stream is not readable.</exception>
+        /// <exception cref="ArgumentNullException">The configuration is null.</exception>
+        /// <exception cref="ArgumentNullException">The data is null.</exception>
+        /// <exception cref="NotSupportedException">The data is not readable.</exception>
         /// <returns>
         /// The <see cref="IImageInfo"/> or null if suitable info detector is not found.
         /// </returns>
         public static IImageInfo Identify(Configuration configuration, byte[] data, out IImageFormat format)
```

Comment:
Can decoders throw `InvalidImageContentException` in `Identify`?

Reply:
Yeah, they can hit a bad value in the image header.